### DISCUSSION TITLE
[询问] 多账号支持？临时停止个别账号？

### DISCUSSION THREAD

**Discussion by chzhong** [NapNeko/NapCatQQ#625] [Ideas]
NapCatQQ 是否要考虑支持多账号运行

目前已经按用 Docker 正常启动并连接了 onebot，现在想添加另一个账号，根据搜索得知似乎无法通过 WebUI 完成，因此想询问：
1、如何添加另外的账号？添加 `./config/onebot11_xxxx.json` 其中 `xxxx` 为新的 QQ 号可行吗？或者删掉浏览器缓存用新QQ扫码登录？
2、续上，如果支持多账号，能否考虑支持控制这些账号按需上线？

实际场景中有同时运行多个号以及临时关闭某个号的需求。既然已经有了 WebUI 希望作者可以考虑一下方便配置。

**Comment by lyjjl** [comment]
目前我找到的办法是同时开多个docker（？？？）
但是我这个机器本来就是极限性能运行的，无法支持我改用docker部署，该方法没有确认过。
似乎是因为本机安装的话文件只有一份，多个号同时登陆一旦体量大了会导致抢文件出现问题所以禁止多账号了。
不过我记得NC的运行方式不是非常依赖QQ客户端来着（？
似乎是有支持的可能和必要

**Reply by lyjjl** [reply]
删除浏览器缓存是不可行的做法
直接通过napcat start 添加已经被禁止，尚不清楚是否可以通过同时将多个账号设为开机自启动来同时登陆多账号
